firefox_puppeteer/ui/toolbars.py

```diff
@@ -315,6 +315,22 @@ def is_open(self):
         return self.popup.get_attribute('state') == 'open'
 
     @property
+    def is_complete(self):
+        """Returns when this popup is open and search results are complete.
```

Comment:
I would not mention `search results` but better `autocomplete results`. Search will confuse here because of the shown search engine entries at the end.